templates/vue/src/components/Lightbox/media/VideoMedia/H5PVideoMedia.vue

```diff
@@ -131,8 +131,9 @@ export default {
         this.settingsLastUpdated = currTimestamp
       }
     },
-
     setFrameDimensions() {
+      // Switching to aspect ratio method for TYDE only
```

Comment:
The aspect ratio method should be something that we can do for all videos through a new Tapestry setting and then be able to override per video. We should probably add this feature for all videos that use an iframe (e.g. H5P, YouTube, Kaltura) so that the author can set the aspect ratio and that would cause this code here to not run.